core/examples/generated/lib/railjson_generator/physical_approximations.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass, field
+from typing import List
+
+import json
+from railjson_generator.schema.infra.range_elements import TrackRange
+
+from schemas import physical_approximations
+
+
+@dataclass
+class ElectricalProfileTest:
```

Comment:
how about `ExperimentalElectricalProfile`?
I also believe the railjson spec lacks this prefix

Reply:
That was a typo, I shouldn't have commited that